packages/storybook/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "storybook",
```

Comment:
In addition to the final name we give to this package I would also like to add a prefix. In the end the name would look like `@small-advertisments/<name-of-ui>`

Reply:
Yes, this part makes sense too. I changed the name already. I couldn't think of a better name, we can change it later or if you have a better suggestion.